backend/src/components/nmfApplication.js

```diff
@@ -19,22 +19,33 @@ function mapNMFApplicationObjectForBack(data) {
   data.remoteCommunities = data.remoteCommunities ? ((data.remoteCommunities === "Yes") ? 1 : 0) : null;
   return new MappableObjectForBack(data, NMFApplicationMappings).toJSON();
 }
-  
-async function getNMFApplication(req, res) {
-  let query = `ccof_rfi_pfi_nmfs?$filter=(_ccof_applicationccfri_value eq ${req.params.ccfriId})`;
+
+async function getNmfApplicationByCcfriId(ccfriId) {
+  let query = `ccof_rfi_pfi_nmfs?$filter=(_ccof_applicationccfri_value eq ${ccfriId})`;
   log.info('GET NMF query ' + query);
+
+  const response = await getOperation(query);
+  console.log('response: ', minify(response.value));
```

Comment:
I think these logs have been here for a while... but I think we should take them out since we are making changes in this file anyways.

Reply:
Removed